packages/viewer/src/interactions/interactionApi.ts

```diff
@@ -191,6 +195,27 @@ export class InteractionApi {
     return this.currentCamera != null;
   }
 
+  /**
+   * Scales the provided `pixelThreshold` by the device's pixel ratio
+   * and the configured `coarsePointerThresholdScale` value if the
+   * primary pointer input is coarse (touch, stylus) or the configured
+   * `finePointerThresholdScale` value if the primary pointer input is
+   * fine (mouse, trackpad).
+   *
+   * @param pixelThreshold - The pixel threshold to scale.
+   * @returns The scaled pixel threshold.
+   */
+  public scalePixelThreshold(
+    pixelThreshold: number,
+    isTouch?: boolean
+  ): number {
+    const scaleFactor = this.isCoarseInputDevice(isTouch)
+      ? this.getConfig().coarsePointerThresholdScale
+      : this.getConfig().finePointerThresholdScale;
+
+    return pixelThreshold * window.devicePixelRatio * scaleFactor;
```

Comment:
Sorry, to clarify, I think it'd be easier to understand if there was just a simple threshold for fine and course pointers and no scale factor. In other words, I can configure a threshold and and it scales by the devices DPI. Having an additional scale factor on top of that adds an extra layer of complexity and doesn't give me a way to simply adjust the pixel threshold.

Reply:
I see, I think in the ticket we specifically wanted the ability to configure mouse/touch thresholds separately. Are you thinking that'd no longer be needed?

Reply:
How about something like this?

```ts
    const pixelThreshold = this.isCoarseInputDevice(isTouch)
      ? this.getConfig().coarsePointerThreshold
      : this.getConfig().finePointerThreshold;

    return pixelThreshold * window.devicePixelRatio;
```

Reply:
So just to make sure I'm following, with this change, we'd essentially remove all the places where previously we had something like `Point.distance(first, second) >= 2` and replace those with `Point.distance(first, second) >= api. pixelThreshold()`? Essentially to get rid of the idea of "scaling" the thresholds we already have, and instead define your own?

Reply:
Yep